### DISCUSSION TITLE
Any ETA for signature insights to be in production?

### DISCUSSION THREAD

**Discussion by hoshiyari420** [MetaMask/snaps#2686] [Q&A] [answered]
We have been working on a snap that helps provide signature insights in better manner. We are able to develop MVP on Metamask flask. Any approx. timeline for signature insights being rolled out in future updates would be really helpful for us.

**Comment by roo-shy** [comment, marked as answer]
It'll be available in `12.5.0` approx end of september

**Reply by Montoya** [reply]
Thank you @roo-shy! Just to give an update on this, 12.5 will go out sometime in October (this month), it was delayed a bit.
